react/Avatar/index.jsx

```diff
@@ -59,6 +60,7 @@ export const Avatar = ({
         text ? styles['c-avatar--text'] : '',
         image ? styles['c-avatar--image'] : '',
         disabled ? styles['c-avatar--disabled'] : '',
+        ghost ? styles['c-avatar--ghost'] : '',
```

Comment:
nit:
is there a test to check this class is present if ghost is present? 
is there a test to check this class is not present when if ghost is missing?